lightstep/propagation.py

```diff
@@ -11,10 +11,15 @@ class LightStepFormat(object):
     These static constants are intended for use in the Tracer.inject() and
     Tracer.extract() methods. E.g.,
 
-        tracer.inject(span.context, LightStepFormat.LIGHTSTEP_BINARY, lightstep_carrier)
+        tracer.inject(
+            span.context, LightStepFormat.LIGHTSTEP_BINARY, lightstep_carrier
+        )
```

Comment:
This style change is not needed (usually, when we want to make style changes, they can come in a separated PR)

Reply:
Ok, rolling it back.

Reply:
Fixed